### ISSUE TITLE
dsc_extractor can't extract symbol files from iOS 15.0

### ISSUE THREAD

**Issue by envestcc** [Zuikyo/iOS-System-Symbols#29]
i have downloaded [iOS 15.0 fireware](https://updates.cdn-apple.com/2021SummerSeed/fullrestores/071-75926/EAF13BB1-BC61-4554-BA5C-AD5F5FEA6ED6/iPhone13,4_15.0_19A5307g_Restore.ipsw).
i followed the steps. when i run dsc_extractor , the result is as bellows:
> dyld_shared_cache_extract_dylibs_progress() => 0

it seems to be not found any symbol files.

**information of my mac**
os version: 11.0.1
xcode: Xcode 12.5.1 Build version 12E507

**Comment by Aashima89** [comment]
yeah same issue for me. Also, if we open the dmg file now (Step 6) it leads to these 3 files
dyld_shared_cache_arm64e
dyld_shared_cache_arm64e.1
dyld_shared_cache_arm64e.symbols

and accordingly I guess the parsing needs to change

**Comment by zhichicao** [comment]
I update Xcode to Xcode 13 and i extract it successfully. Seems that it need a new bundle file in Xcode 13:
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/usr/lib/dsc_extractor.bundle

**Comment by kylescf2** [comment]
> I made a supplement to symol files here. https://github.com/CXTretar/iOS-System-Symbols-Supplement

thanks，good job
